packages/zosmf/__tests__/__system__/cli/list/systems/zosmf.list.systems.system.test.ts

```diff
@@ -0,0 +1,124 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { runCliScript, stripNewLines } from "../../../../../../../__tests__/__src__/TestUtils";
+import { TestEnvironment } from "../../../../../../../__tests__/__src__/environment/TestEnvironment";
+import { ITestEnvironment } from "../../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { TestProperties } from "../../../../../../../__tests__/__src__/properties/TestProperties";
+import { ITestSystemSchema } from "../../../../../../../__tests__/__src__/properties/ITestSystemSchema";
+import { IO } from "@brightside/imperative";
+
+let testEnvironment: ITestEnvironment;
+let systemProps: TestProperties;
+let host: string;
+let port: number;
+let user: string;
+let pass: string;
+
+
+describe("zosmf list systems", () => {
+
+    // Create the unique test environment
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            tempProfileTypes: ["zosmf"],
+            testName: "zos_list_systems"
+        });
+
+        systemProps = new TestProperties(testEnvironment.systemTestProperties);
```

Comment:
Hi, 

You will have to update this slightly because of #332 

use host = testEnvironment.systemTestProperties.zosmf.host; instead of this  new TestProperties syntax